packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -215,10 +223,14 @@ class Group extends Component {
 
     render() {
         const { isExpanded, filterBy } = this.state;
-        const { groups, items, placeholder, className, selected, isFilterable, isDisabled, onFilter } = this.props;
-
+        const { groups, items, placeholder, className, selected, isFilterable, isDisabled, onFilter, onShowMore } = this.props;
         const filterItems = items || groups;
 
+        const showMore = {
+            type: groupType.button,
+            variant: 'link',
+            items: [{ label: 'Show more', type: groupType.button, onClick: onShowMore, isPersistentAction: true }]
```

Comment:
Can we have this changable? Like default to `Show more`, but if app consumer decide that it should show `Show more 10 items` we should give them the option to do that.